docs/rulesets.md

```diff
@@ -140,14 +147,14 @@ Rules are highly configurable. There are only few required parameters but the op
   </tbody>
 </table>
 
-### Ruleset validation
+## Ruleset Validation
 
 We use JSON Schema & AJV to validate your rulesets file and help you spot issues early.
 
 **Example output**
 
 ```bash
-spectral lint some-oas.yaml --ruleset acme-company.json
+$ spectral lint some-oas.yaml --ruleset acme-company.json
```

Comment:
remove `$` 

they get copied and result in annoying bash errors for users.

Reply:
most Markdown linters recommend this when commands and output are recommended. My IDE is recommending this right now. 

If it was a command I was expecting people to copy, like a getting started / installation section, that would be different (and it would not be mixing commands and output either so my markdown linter would not shout at me). :)

Reply:
Updated: typos ☝️